src/app/components/Forms/EditForm.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import EditForm from './EditForm'
+
+export default {
+  title: 'Component/EditForm',
+  component: EditForm,
+}
+
+export const Need = (): JSX.Element => (
```

Comment:
I don't understand `Need`, `Topic` and `Proposal` :).

Reply:
The edit form can be used to edit needs, topics and proposals